graal/backends/core/colic.py

```diff
@@ -101,13 +105,15 @@ def fetch(self, category=CATEGORY_COLIC_NOMOS, paths=None,
     def metadata_category(item):
         """Extracts the category from a Code item.
 
-        This backend generates two types of item which can be:
-        'code_license_nomos' or 'code_license_scancode'.
+        This backend generates three types of item which can be:
```

Comment:
Would you think to change this comment with: `This backend generates the following types of item:`. In this way we don't have to remember of modifying this text if more categories are added.

Reply:
Yes. Thanks for the suggestion.
I'll update this, Thanks